### PR TITLE
Adding workflow to deploy a Bastion host.  Updating docs.

### PR DESCRIPTION
**Pull request by devopsdina**
- Adding workflow to deploy a Bastion host
- Updating documentation

### CODE REVIEW

**Comment by devopsdina** [comment]
Please don't merge this!  Please comment, approve it, or reject it!  

I will "hit merge" once it is approved so I can edit the final commit message.

**Review by giofontana — APPROVED** [review]
Nice adds! Approved!

**Pull request merged**
